Meadowlark-js/services/meadowlark-aws-lambda/resources/functions.yml

```diff
@@ -38,7 +38,7 @@ metaed:
   events:
     - http:
         method: get
-        path: /metaed/{projectAbbreviation}
+        path: /metaed
```

Comment:
Only use of path parameterization. This was for the earliest of Meadowlark testing, that we were successfully parsing and loading a MetaEd project.